pkg/bufferdecoder/eventsreader.go

```diff
@@ -13,40 +13,6 @@ import (
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
-// argType is an enum that encodes the argument types that the BPF program may write to the shared buffer
-// argument types should match defined values in ebpf code
-type ArgType uint8
-
-const (
-	noneT ArgType = iota
-	intT
-	uintT
-	longT
-	ulongT
-	offT
-	modeT
-	devT
-	sizeT
-	pointerT
-	strT
-	strArrT
-	sockAddrT
-	bytesT
-	u16T
-	credT
-	intArr2T
-	uint64ArrT
-	u8T
-	timespecT
-)
```

Comment:
As mentioned in a previous commit comment, I think this enum (for that matter, DecodeAs enum is the same as this one) should be kept here since it is the business of the decoder, and not of any other part in the code, and more specifically' shuoldn't be part of the types packages

Reply:
See above.